buildSrc/src/main/kotlin/Version.kt

```diff
@@ -1,17 +1,17 @@
 object Version {
     // Kotlin
-    const val kotlin = "1.3.61"
-    const val stdLib = "1.3.61"
+    const val kotlin = "1.4.10"
+    const val stdLib = "1.4.10"
 
     // Libraries
     const val jackson = "2.10.0"
     const val slf4j = "1.7.30"
     const val jUnit = "5.6.0"
     const val jetty = "9.4.3.v20170317"
 
-    const val ktor = "1.3.1"
-    const val serializationRuntime = "0.14.0"
-    const val coroutinesCore = "1.3.3"
+    const val ktor = "1.4.0"
+    const val serializationRuntime = "1.0.0"
+    const val coroutinesCore = "1.3.9"
```

Comment:
Why not 1.4.0+?
There are breaking changes in 1.4.0, it's easier to pass through it now, while we don't have a big coroutines' codebase

Reply:
Ktor dependencies use coroutines 1.3.9, omit 1.3.9 to 1.4.1 in runtime may bring troubles.